docs/source/roles/researcher/user_guide.md

```diff
@@ -722,12 +722,16 @@ The contents of `/shared/` will be **identical** on all SRDs in your SRE.
 The `/scratch/` folder should be used for any work-in-progress that isn't ready to share yet.
 Although everyone in your group will have **read-and-write access**, you can create your own folders inside `/scratch` and choose your own permissions for them.
 
+```{caution}
+You should not use `/scratch/` for long-term storage as it can be reset at any time without warning (e.g. when the VM is restarted).
```

Comment:
```suggestion
You should not use `/scratch/` for long-term storage as it can be reset at any time without warning (_e.g._ when the VM is restarted).
```